packages/snaps-simulator/jest.config.js

```diff
@@ -2,13 +2,15 @@ const deepmerge = require('deepmerge');
 
 const baseConfig = require('../../jest.config.base');
 
+delete baseConfig.transform;
+
 module.exports = deepmerge(baseConfig, {
   coverageThreshold: {
     global: {
-      branches: 72.5,
-      functions: 79.28,
-      lines: 89.37,
-      statements: 89.16,
+      branches: 56.95,
```

Comment:
Why did this drop so badly?

Reply:
We didn't collect coverage from `.tsx` files previously. A lot of files in the simulator aren't tested.